hackstar.py

```diff
@@ -101,6 +102,108 @@ async def shazam(audio_file):
     return title, artist, release_date, cover
 
 
+def apply_loudnorm_filter(input_file, output_file):
+    """
+    Apply 2-pass loudnorm filter for audio normalization and compression.
+    
+    Args:
+        input_file (str): Path to input audio file
+        output_file (str): Path to output audio file
+    
+    Returns:
+        bool: True if processing succeeded, False otherwise
+    """
+    try:
+        logger.info("Starting loudnorm processing for %s", input_file)
+        
+        # Convert to absolute paths to avoid path issues
+        abs_input = os.path.abspath(input_file)
+        abs_output = os.path.abspath(output_file)
+        
+        # First pass: Detect current loudness levels
+        ffprobe_command = [
+            "ffprobe",
+            "-f", "lavfi",
+            "-hide_banner",
+            "-i", f"amovie={abs_input},loudnorm=print_format=json[out]",
+            "-loglevel", "info"
+        ]
+        
+        logger.debug("Running first pass (loudness detection): %s", " ".join(ffprobe_command))
+        result = subprocess.run(
+            ffprobe_command,
+            text=True,
+            capture_output=True
```

Comment:
We should fail and throw an exception if there is an error:
```suggestion
            capture_output=True,
            check=True
```